test/attr/accessibility_proto.swift

```diff
@@ -1,3 +1,4 @@
+// RUN: %target-typecheck-verify-swift -package-name myPkg
```

Comment:
Adding this back (removed in the previous PR); changes in this file are needed to match the original (before the PR)